modules/date.js

```diff
@@ -0,0 +1,8 @@
+import { DateTime } from '../node_modules/luxon/build/es6/luxon.js';
+
```

Comment:
- [ ] You may consider using the minified version from [jsdelivr](https://cdn.jsdelivr.net/npm/luxon@2.0.2/build/global/luxon.min.js) because the way you're having to import it while explicitly doing `../node_modules` is not ideal, and I noticed that if you do `import { DateTime } from 'luxon'` your app stops working 🙏